### PR TITLE
Code review request

### PR DESCRIPTION
**Pull request by Alan-Liang-1997**
Attempted to fix the pull request issue for Prac 5. I was having issues with Github and making the extra branch but i believe everything has been fixed for everything after Prac 5. I am not sure why  the changes are showing up with so much edits.

### CODE REVIEW

**Comment by Alan-Liang-1997** [comment]
Hi @GregMcLindon @lindsaymarkward

This is the new pull request for Prac 5. I have made the changes and fixes you highlighted in the other code review request. It is much easier to read the code by just viewing the file instead of observing the additions/removals. I was having issues getting what i wanted to appear on the code review request.

**Comment by lindsaymarkward** [comment]
Please remove the `.idea` files from your repo - see Prac 04 for details (you'll need to use the command line).

File: prac_05/state_names.py [hunk 1]
```diff
@@ -1,21 +1,18 @@
...
-# TODO: Reformat this file so the dictionary code follows PEP 8 convention
 STATE_NAMES = {"QLD": "Queensland", "NSW": "New South Wales", "NT": "Northern Territory", "WA": "Western Australia",
                "ACT": "Australian Capital Territory", "VIC": "Victoria", "TAS": "Tasmania"}
-# print(STATE_NAMES)
 
-state = input("Enter short state: ").upper()
-while state != "":
-    if state in STATE_NAMES:
-        print(state, "is", STATE_NAMES[state])
+full_state = input("Enter short state: ").upper()
```
Comment (lindsaymarkward) [inline prac_05/state_names.py]: Why would `full_state` be a good name for "short state"?

File: prac_05/state_names.py [hunk 2]
```diff
@@ -1,21 +1,18 @@
...
+while full_state != "":
+    if full_state in STATE_NAMES:
+        print(full_state, "is", STATE_NAMES[full_state])
     else:
         print("Invalid short state")
-    state = input("Enter short state: ").upper()
+    full_state = input("Enter short state: ").upper()
```
Comment (GregMcLindon) [inline prac_05/state_names.py]: Think I'm partly to blame for this one saying 'state_long' might get confused for number type 'Long'.
Maybe use 'state_short_desc' and 'state_long desc'.
And yes as Lindsey said... this should be:
       `state_short_desc = input("Enter short state: ").upper() `

File: prac_05/state_names.py [hunk 3]
```diff
@@ -1,21 +1,18 @@
...
 
-for short_name ,state in STATE_NAMES.items():
-    print("{:<3} is {:<10}".format(short_name ,state))
\ No newline at end of file
+for short_state, full_state in STATE_NAMES.items():
```
Comment (GregMcLindon) [inline prac_05/state_names.py]: You've got the short/long correct here so only update the variable names (not the order you've got them listed)

**Pull request merged**